proto/osmosis/swaprouter/v1beta1/tx.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+package osmosis.swaprouter.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v12/x/swaprouter/types";
+
+service Msg {
+  rpc SwapExactAmountIn(MsgSwapExactAmountIn)
+      returns (MsgSwapExactAmountInResponse);
+  rpc SwapExactAmountOut(MsgSwapExactAmountOut)
+      returns (MsgSwapExactAmountOutResponse);
+}
+
+// ===================== MsgSwapExactAmountIn
+message SwapAmountInRoute {
+  uint64 pool_id = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  string token_out_denom = 2
+      [ (gogoproto.moretags) = "yaml:\"token_out_denom\"" ];
+}
+
+message MsgSwapExactAmountIn {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  repeated SwapAmountInRoute routes = 2 [ (gogoproto.nullable) = false ];
```

Comment:
Do we support multi hop or multi routing for concentrated pools?

Reply:
Yes, these are wrappers arount multihop